store/src/main/java/com/expedia/alertmanager/store/App.java

```diff
@@ -51,7 +53,12 @@ public static void main(String[] args) throws Exception {
 
     private static StoreConfig loadConfig(String[] args) throws IOException {
         File configFile = null;
-        if (args != null && args.length == 1 && !args[0].isEmpty()) {
+        Set<String> allowedconfigFiles = new HashSet<String>();
```

Comment:
can be moved to one line `new HashSet(Arrays.asList("application.yml", "application-dev.yml", "/tmp/application.yaml"))`